### ISSUE TITLE
Expo Error: Cannot find module './debug'

### ISSUE THREAD

**Issue by bugs181** [visionmedia/debug#679]
Just came back to an Expo project after about a month and now receiving an odd error. As far as I know, nothing in the repo should have been updated.

```
[17:07:14] (node:11476) UnhandledPromiseRejectionWarning: Error: Cannot find module './debug'
[17:07:14]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
[17:07:14]     at Function.Module._load (internal/modules/cjs/loader.js:513:25)
[17:07:14]     at Module.require (internal/modules/cjs/loader.js:643:17)
[17:07:14]     at require (internal/modules/cjs/helpers.js:22:18)
[17:07:14]     at Object.<anonymous> (/Users/Levi/.../app/node_modules/debug/src/node.js:14:28)
[17:07:14]     at Module._compile (internal/modules/cjs/loader.js:707:30)
[17:07:14]     at Module._compile (/Users/Levi/.../app/node_modules/.registry.npmjs.org/pirates/4.0.0/node_modules/pirates/lib/index.js:83:24)
[17:07:14]     at Module._extensions..js (internal/modules/cjs/loader.js:718:10)
[17:07:14]     at Object.newLoader [as .js] (/Users/levi/.../app/node_modules/.registry.npmjs.org/pirates/4.0.0/node_modules/pirates/lib/index.js:88:7)
[17:07:14]     at Module.load (internal/modules/cjs/loader.js:605:32)
[17:07:14] (node:11476) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
[17:07:20] Metro Bundler process exited with code 0
```

To temporarily fix the issue change line `14` in `node.js` to:
```
exports = module.exports = require('../dist/debug');
```

In addition to the change, I replaced `npm` with `pnpm` to reduce wasted dependency space.

Potentially related: https://github.com/visionmedia/debug/issues/228 https://github.com/visionmedia/debug/issues/606 https://github.com/visionmedia/debug/issues/390

**Comment by Qix-** [comment]
Please provide:

- package.json
- node version
- the version of debug that's actually preset (`cat node_modules/debug/package.json`)
- are you using lockfiles?
- did you delete your lockfile/`node_modules` and re-install?

**Comment by bugs181** [comment]
Apologies, was an older version of this package. I've since reinstalled and line 14 from my previous issue does not apply anymore. You may close or keep open for posterity sake in case anyone else has this issue with Expo. The version in history appears to be `2.6.9`. This issue is fixed in `4.1.0` with Expo `31.0.4` but Expo users on older `< 31` may need to be aware of the upgrade.

```
    "expo": "^29.0.0",
    "react": "16.3.1",
```
